phantoms/MR_XCAT_qMRI/sim_ivim_sig.py

```diff
@@ -261,7 +273,7 @@ def XCAT_to_MR_DCE(XCAT, TR, TE, bvalue, D, f, Ds, b0=3, ivim_cont = True):
         73: 'Pancreas tumor',
     }
     ###############################################################################
-    np.random.seed(42)
+    np.random.seed(41)
```

Comment:
Seed change?

Reply:
This is now fixed. Changed it to test something :).